install/roles/server/tasks/uninstall_docker_containers.yml

```diff
@@ -34,7 +34,7 @@
   when:
     - have_qpc_db_docker_container
 
-- name: Set variable indicating whether any QPC docker containers found
+- name: Set variable indicating whether any quipucords docker containers found
```

Comment:
change to `Set variable indicating whether any Quipucords docker containers are present`